src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx

```diff
@@ -73,12 +76,12 @@ const backend = defineBackend({
 
 // extract L1 CfnUserPool resources
 const { cfnUserPool } = backend.auth.resources.cfnResources;
-// use CDK's `addPropertyOverride` to modify properties directly
-cfnUserPool.addPropertyOverride("Schema", [
+// modify cfnUserPool schema directly
+cfnUserPool.schema = [
```

Comment:
```suggestion
cfnUserPool.schema = [
   ...(cfnUserPool.schema || [])
```

we can spread the existing schema to justify removing the callout above. despite the property being optional when we know it will always exist, we'll also need the fallback value to an empty array

Reply:
Since type of schema is `IResolvable | (IResolvable | CfnUserPool.SchemaAttributeProperty)[] | undefined` we would have to cast it otherwise Typescript complains that `IResolvable` is not iterable:
```ts
cfnUserPool.schema = [
  ...(cfnUserPool.schema as any[] || [])
```

Reply:
ah that is unfortunate. what about
```ts
if (Array.isArray(cfnUserPool.schema)) {
  cfnUserPool.schema.push(/* ... */)
}
```